raiden_contracts/deploy/__main__.py

```diff
@@ -122,6 +137,8 @@ def main(**kwargs):
             txn_wait,
         )
 
+        time.sleep(2)
```

Comment:
Why is this needed? We use `check_succesful_tx`, that waits for the transaction to be mined.